RELEASE.md

```diff
@@ -2,6 +2,103 @@
+  as of writing this) and make sure it works:
+  - [ ] Create a new development environment with Poetry.
+  - [ ] Build the container image and ensure the development environment uses
+    the new image.
+  - [ ] Run the Dangerzone tests.
+  - [ ] Create a .deb package and install it system-wide.
+  - [ ] Test some QA scenarios (see [Scenarios](#Scenarios) below).
+- [ ] Create a test build in the most recent Fedora platform (Fedora 37 as of
+  writing this) and make sure it works:
+  - [ ] Create a new development environment with Poetry.
+  - [ ] Build the container image and ensure the development environment uses
+    the new image.
+  - [ ] Run the Dangerzone tests.
+  - [ ] Create an .rpm package and install it system-wide.
+  - [ ] Test some QA scenarios (see [Scenarios](#Scenarios) below).
+- [ ] Run only the CI tests in the rest of the Linux platforms:
+  - [ ] Create a new development environment with Poetry.
+  - [ ] Build the container image and ensure the development environment uses
+    the new image.
+  - [ ] Run the Dangerzone tests.
+
+### Scenarios
+
+#### 1. Dangerzone correctly identifies that Docker/Podman is not installed
+
+Temporarily hide the Docker/Podman binaries, e.g., rename the `docker` /
+`podman` binaries to something else. Then run Dangerzone. Dangerzone should
+prompt the user to install Docker/Podman.
+
+#### 2. Dangerzone correctly identifies that Docker is not running
+
+_(Only for MacOS / Windows)_
+
+Stop the Docker Desktop application. Then run Dangerzone. Dangerzone should
+prompt the user to start Docker Desktop.
+
+#### 3. Dangerzone successfully installs the container image
+
+Remove the Dangerzone container image from Docker/Podman. Then run Dangerzone.
+Danerzone should install the container image successfully.
+
+#### 4. Dangerzone retains the settings of previous runs
+
+Run Dangerzone and make some changes in the settings (e.g., change the OCR
+language, toggle whether to open the document after conversion, etc.). Restart
+Dangerzone. Dangerzone should show the settings that the user chose.
+
+#### 5. Dangerzone's system tray is functional
+
+_(Only for MacOS / Windows)_
+
+Go to the system tray of Dangerzone at the bottom panel and ensure that "New
+Window" opens a new window, and "Quit" closes all windows.
+
+#### 6. Dangerzone reports failed conversions
+
+Change a document's filetype to something else that Dangerzone handles. Run
+Dangerzone and convert the document. Dangerzone should fail gracefully, by
+reporting that the operation failed, and showing the last error message.
+
+#### 7. Dangerzone succeeds in converting multiple documents
+
+Run Dangerzone against a list of documents, and tick all options. Ensure that:
+* Conversions start in parallel.
+* Conversions are completed successfully.
+* Conversions show individual progress.
+* The resulting files open with the PDF viewer of our choice.
+* OCR seems to have detected characters in the PDF files.
```

Comment:
That's an interesting idea. In this case though, we would need to do a second conversion with a different OCR language than English, since we can't choose the language per document. And that makes me a bit hesitant, as our list of checks is already long :-/.

Reply:
OK. Then let's not implement this, but consider making in the future an automated test to detect this.